apps/dashboard/src/components/workflow-editor/test-workflow/test-workflow-tabs.tsx

```diff
@@ -53,8 +53,10 @@ export const TestWorkflowTabs = ({ testData }: { testData: WorkflowTestDataRespo
           <>
             <ToastIcon variant="default" />
             <div className="flex flex-col gap-2">
-              <span className="font-medium">Test workflow triggered successfully</span>
-              <span className="text-foreground-600">{`Test workflow ${workflowSlug} was triggered successfully`}</span>
+              <span className="font-medium">The test trigger was completed</span>
+              <span className="text-foreground-600">
+                Workflow ${workflow?.workflowId} was triggered successfully from the Dashboard.
```

Comment:
We are following the pattern of wrapping resource names with strong styling in other places, we could do the same here

```suggestion
                Workflow <strong>${workflow?.workflowId}</strong> was triggered successfully from the Dashboard.
```

Reply:
I saw the link to change from using slug to workflowId.

Don't we think showing the workflow name would be more relevant to the Dashboard user?

Reply:
I saw the link to change from using slug to workflowId.

Don't we think showing the workflow name would be more relevant to the Dashboard user?

Reply:
If we go with the slug, adding the strong emphasis probably isn't needed